shared/checks/oval/installed_env_has_systemd_package.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="inventory"
+  id="installed_env_has_systemd_package" version="1">
+    <metadata>
+      <title>Package systemd is installed</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Checks if package systemd is installed.</description>
+      <reference ref_id="cpe:/a:systemd" source="CPE" />
+    </metadata>
+    <criteria>
+      <criterion comment="Package systemd is installed" test_ref="test_env_has_systemd_installed" />
+    </criteria>
+  </definition>
+
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="system has package systemd installed" id="test_env_has_systemd_installed" version="1">
+    <linux:object object_ref="obj_env_has_systemd_installed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_env_has_systemd_installed" version="1">
+    <linux:name>systemd</linux:name>
+  </linux:rpminfo_object>
```

Comment:
This also can make some rules `notapllicable` on non-RedHat products

Reply:
If they don't have systemd, is that a problem?